pkg/coremodel/playlist/coremodel.cue

```diff
@@ -0,0 +1,49 @@
+package playlist
+
+import (
+	"github.com/grafana/thema"
+)
+
+thema.#Lineage
+name: "playlist"
+seqs: [
+	{
+		schemas: [
+			{//0.0
+				// Unique numeric identifier for the playlist.
+				// TODO: Isolate or remove identifiers local to a Grafana instance
+				id?: int64 @reviewme()
+				// Unique playlist identifier. Generated on creation, either by the
+				// creator of the playlist of by the application.
+				uid?: *"" | string & len(uid)<=80 @reviewme()
+				// Name of the playlist.
+				name?: *"" | string & len(name)<=255 @reviewme()
+				// Interval
+				// TODO: Figure out the type of this, and validate that it's how long between shifts between items in the playlist.
+				interval?: *"5m" | string & len(interval)<=255 @reviewme()
+				// Playlist item
+				items?: #Items
+
+				///////////////////////////////////////
+				// Definitions (referenced above) are declared below
+
+				#Items: {
+					// Unique numeric identifier for the item references by the playlist.
+					// TODO: Isolate or remove identifiers local to a Grafana instance
+					id?: int64 @reviewme()
+					// Unique identifier for the item references by the playlist.
+					uid?: string
+					// Type of the item.
+					type?: "dashboard_by_id" | "dashboard_by_tag" | "dashboard_by_uid"
+					// Title of the playlist item.
+					title?: string
```

Comment:
I don't think we even need title!   It is used in the UI to show the name of the dashboard... but that should really just be loaded from the UID:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/705951/186063954-e44fb380-a805-4dd2-9dfd-818c539f6644.png">

Reply:
I think deduplication of the title would be nice, since names change and dashboards move.

But I'm not going to do that for now, I somewhat want to keep the models close to each other to reduce the complexity of experiments.